model/outgoing_webhook.go

```diff
@@ -199,7 +201,9 @@ func (o *OutgoingWebhook) PreUpdate() {
 }
 
 func (o *OutgoingWebhook) HasTriggerWord(word string) bool {
-	if len(o.TriggerWords) == 0 || len(word) == 0 {
+	if len(o.TriggerWords) == 0 {
+		return true
```

Comment:
This makes the behavior of these two functions really unintuitive given the names (I wouldn't expect `OutgoingWebhook{}.HasTriggerWord("foo")` to return `true`.). It seems like your other changes should fix the issue, so is this change actually necessary?